files/scripts/benchmark_job.sh

```diff
@@ -54,6 +65,10 @@ x86_64*)
     node_type=x86
     machine_type=x86_64
     ;;
+*)
+    echo "UNKNOWN MACHINE TYPE, Exiting..."
```

Comment:
Don't need to be all caps.
Also, echo what's the node, in quotes, so that the user knows it's either empty or where it came from.

ex: echo "Node '${node}' not found"